src/AutoRest.CSharp/Properties/launchSettings.json

```diff
@@ -468,6 +472,10 @@
       "commandName": "Project",
       "commandLineArgs": "--standalone $(SolutionDir)\\test\\TestProjects\\ResourceRename\\Generated"
     },
+    "roundtrip-basic": {
```

Comment:
can we call it 'roundtrip-basic-cadl' to seperate it from swagger input?  Same for other new cases.

Reply:
I'd prefer to have `cadl` as prefix

Reply:
I would prefer prefix too.